components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -242,8 +242,10 @@ be lost if SCREAM_HACK_XML is not enabled.
 
     <!-- ML correction -->
     <mlcorrection inherit="atm_proc_base">
-      <ML_model_path type="string" doc="Path to pre-trained ML model"/>
+      <ML_model_path_tq type="string" doc="Path to pre-trained ML model for temperature and specific humidity"/>
```

Comment:
Is there a reason we need to split the variables into two ML models.

Reply:
Yes, they are trained in separate models because the inputs are not the same. The team has in the past tried to train one model for Tquv but did not have good success. Having them separate also makes it easier to test Tq vs. Tquv. Later on, we will add another one for surface fluxes.